src/Controllers/login.controller.ts

```diff
@@ -0,0 +1,35 @@
+import { Request, Response } from 'express';
+import { checkLoginData } from 'Services/login.service';
+import { User } from 'Types/User';
+import jswonwebtoken from 'jsonwebtoken';
+
+export const loginController = async (
+    req: Request,
```

Comment:
I think that you could add a body type here since Request is a generic type

Reply:
resolved